lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js

```diff
@@ -53,43 +54,31 @@ Y.namespace('M.atto_kalturamedia').Button = Y.Base.create('button', Y.M.editor_a
                 callback: this._kalturamedia
             });
     },
-    _kalturamedia: function(){
+    _kalturamedia: function() {
         this._currentSelection = this.get('host').getSelection();
         if (this._currentSelection === false) {
             return;
         }
-        
-        var dialogue = this.getDialogue({
-            headerContent: M.util.get_string('popuptitle', COMPONENTNAME),
-            focusAfterHide: true,
-            width: '1112px',
-            focusOnShowSelector: SELECTORS.URLINPUT
-        });
-
-        var iframe = Y.Node.create('<iframe></iframe>');
-        // We set the height here because otherwise it is really small. That might not look
-        // very nice on mobile devices, but we considered that enough for now.
-        iframe.setStyles({
-            height: '600px',
-            border: 'none',
-            width: '100%'
-        });
-        
-        var embedButton = Y.Node.create('<button></button>');
-        embedButton.setAttribute('id', 'KalturaMediaSubmit');
-        embedButton.setAttribute('disabled', 'disabled');
-        embedButton.setHTML(M.util.get_string('embedbuttontext', COMPONENTNAME));
-        embedButton.hide();
-        embedButton.on('click', this.embedItem, this);
-        
-        iframe.setAttribute('src', this._getIframeURL());
-        
-        var containter = Y.Node.create('<div></div>');
-        containter.append(iframe);
-        containter.append(embedButton);
-        // Set the dialogue content, and then show the dialogue.
-        dialogue.set('bodyContent', containter)
-                .show();
+
+        var w = 1200;
+        var h = 700;
+        var dualScreenLeft = window.screenLeft != undefined ? window.screenLeft : screen.left;
+        var dualScreenTop = window.screenTop != undefined ? window.screenTop : screen.top;
+
+        var width = window.innerWidth ? window.innerWidth : document.documentElement.clientWidth ? document.documentElement.clientWidth : screen.width;
+        var height = window.innerHeight ? window.innerHeight : document.documentElement.clientHeight ? document.documentElement.clientHeight : screen.height;
+
+        var left = ((width / 2) - (w / 2)) + dualScreenLeft;
+        var top = ((height / 2) - (h / 2)) + dualScreenTop;
+        var newWindow = window.open(this._getIframeURL(), "Browse and Embed", 'scrollbars=no, width=' + w + ', height=' + h + ', top=' + top + ', left=' + left);
```

Comment:
@muli don't we have translation framework in this plugin ? ("Browse and Embed" string for example)